rpm_spec/manageiq.spec.in

```diff
@@ -47,18 +47,12 @@ pathfix.py -pni "%{__python3} %{py3_shbang_opts}" %{gemset_builddir}
 %build
 cd %{_builddir}
 
-cat <<"EOF" > %{gemset_builddir}/enable
-export APPLIANCE="true"
-export BUNDLE_GEMFILE=%{app_root}/Gemfile
-export GEM_HOME=%{gemset_root}
-export GEM_PATH=%{gemset_root}:$(gem env path)
-export PATH=%{gemset_root}/bin:$PATH
-EOF
-
-cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/evm_production
-export APPLIANCE_SOURCE_DIRECTORY=%{appliance_root}
-export APPLIANCE_TEMPLATE_DIRECTORY=${APPLIANCE_SOURCE_DIRECTORY}/TEMPLATE
-source %{gemset_root}/enable
+cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/production.properties
```

Comment:
Only suggestion is we keep some sort of indicator that this is related to miq in the name.  A file named production.properties is very generic.  @agrare @bdunne Thoughts?

Reply:
sure, you want `evm_production.properties`? we already have `evm.properties`
or we could have `evm_appliance.properties`?

... naming ...

Reply:
`evm_production.properties` seems good

Reply:
would love to remove the evm name - could also do` manageiq_production.properties`

Reply:
Sorry to come in late here and I don't want to just go back and forth on naming, but why have `production` in the name at all?  Seems redundant given this is for an RPM being installed on a MIQ appliance.  Would `/etc/default/manageiq` or `/etc/default/manageiq.properties` would?

Reply:
@agrare I think it's because there is a base properties, and then these are extra properties.

Reply:
So, in discussion we determined that these env vars are "dynamic" based on the productization of the paths, as opposed to the other env vars which are "static".  The static file will be named manageiq.properties and the dynamic file will be named manageiq-productizaton.properties